hedyorg/hedy
# [FIX] Fix ask modal not being shown after editing program

**Description**

I replaced all of the references to the variable `askPromptShown` to reference the variable in the editor object. This caused that when you didn't answer an ask the variable wasn't being set correctly. I also added a couple of test for the ask modal.

**Fixes #4458 **

**How to test**
The cypress should suffice, but you can also try it going to level 1 and typing the program:
 ```
print Hello
ask hello
 ```
then execute that program and write in the editor box without answering the input:
print hello

Now when you run the program again the input should be shown normally.
**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done)


If you're unsure about any of these, don't hesitate to ask. We're here to help!
